src/scenic/syntax/scenic.gram

```diff
@@ -1101,6 +1237,17 @@ try_stmt[ast.Try]:
     | 'try' &&':' b=block ex=except_block+ el=[else_block] f=[finally_block] {
         ast.Try(body=b, handlers=ex, orelse=el or [], finalbody=f or [], LOCATIONS)
      }
+    | 'try' &&':' b=block ex=except_star_block+ el=[else_block] f=[finally_block] {
```

Comment:
We'll have to decide whether we want to allow `except*` (from Python 3.11) in `try-interrupt` statements. Offhand I can't see any use for that, so I'm fine disallowing it as in the current grammar. But then we should probably add a sentence to the end of the documentation for the `try` statement as this is a further difference between Scenic's `try` and Python's.

Reply:
Good call, I've added a note to the docs.